nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -55,14 +54,17 @@ BtNavigator::on_configure(const rclcpp_lifecycle::State & /*state*/)
   self_client_ = rclcpp_action::create_client<nav2_msgs::action::NavigateToPose>(
     client_node_, "NavigateToPose");
 
-  goal_sub_ = rclcpp_node_->create_subscription<geometry_msgs::msg::PoseStamped>(
```

Comment:
Can't think of the reason why in this case the subscription was created on the `rclcpp_node`.

Reply:
Its to enable the goal pose tool in rviz to work, ei `move_base_simple/goal`-esk behavior. That should be readded

Reply:
@SteveMacenski  I didn't remove the goal pose subscription, I just added it to the BtNavigator node itself instead of the `rclcpp_node_`

Reply:
My bad (I'm 0/2 today)